### ISSUE TITLE
Building the sev-host tool

### ISSUE THREAD

**Issue by rrkapitz** [AMDESE/sev-guest#4]
Hi, I tried based on the desc. on docs/cloud-host-setup.md to build the sev-host tools and the following things came up:
1.) The Makefile references sev-host.c which seems not to be in the repo.
2.) The desc. suggests: "apt install libssl-dev uuid-dev". I did this for an updated Ubuntu Ubuntu 20.04.3 LTS which in the other repos for the SEV related topics is fine. However, it seems as openssl-3.0 is needed and not the version libssl 1.1.1f that is part of the distro. The openssl/decoder.h is the important point here.
3.) When using a freshly compiled version of openssl-3.0.1 the build also breaks for:
/opt/openssl/include/openssl/bio.h:562:34: warning: value computed is not used [-Wunused-value]
  562 | # define BIO_flush(b)            (int)BIO_ctrl(b,BIO_CTRL_FLUSH,0,NULL)
      |                                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
(Flag is -Werror)

So nothing dramatic but maybe it helps others.

**Comment by jlarrew** [comment]
Thanks for the report! My .gitignore was accidentally filtering out sev-host.c from my commits. I just pushed commit abe2b641e68c767ad46949f05fe86f64fc951344 to fix the problem.

Regarding OpenSSL, I just pulled the latest tip (fbbe7202eb), and the build works for me. I was previously building against commit d99004fe5de934120765d342586f08d22131b8ed, which also worked. Maybe rebuilding OpenSSL from commit will help?

I'm currently refreshing the documentation in the docs/ directory, so I'll be sure to include the openssl-3.0 dependency.

Thanks!
